scripts/live-connector-e2e/report.py

```
@@ -56,36 +96,165 @@ def load_results(results_dir: Path, *, roots: list[Path] | None = None) -> list[
             continue
         try:
             with path.open(encoding="utf-8") as f:
-                for line in f:
+                for line_number, line in enumerate(f, start=1):
                     line = line.strip()
```

Ruff (0.16.1)

[warning] `for` loop variable `line` overwritten by assignment target

(PLW2901)